File: Cpp/Linked List/linkedListRepresentation.cpp
// Online C++ compiler to run C++ program online
#include <iostream>
using namespace std;

struct Node {
    int data;
    struct Node * next;
};
int main() {
    struct Node * head = new struct Node();
    head->data = 4;
    head->next = NULL;
    struct Node * temp = new struct Node();
    temp->data = 6;
    temp->next = NULL;
    head->next = temp;
    cout<< head->data;
    return 0;
}

File: Java/Linked List/LinkedListRepresentation.java
class Node{
    int data;
    Node next;
}
class Main {
    public static void main(String[] args) {
        Node head = new Node();
        head.data = 4;
        head.next = null;
        Node temp = new Node();
        temp.data = 6;
        temp.next = null;
        head.next = temp;
       System.out.println(head.data);
       System.out.println(head.next.data);
    }
}
